manifests/firewall.pp

```diff
@@ -0,0 +1,87 @@
+# Configure firewall
+class foreman_network::firewall {
+
+  # We don't want to purge rules other than we specify below (could disable ssh etc)
+  if ($foreman_network::configure_firewall_ssh or
+      $foreman_network::configure_firewall_dns or
+      $foreman_network::configure_firewall_dhcp or
+      $foreman_network::configure_firewall_bootp or
+      $foreman_network::configure_firewall_tftp or
+      $foreman_network::configure_firewall_apache or
+      $foreman_network::configure_firewall_puppet) {
+    resources { 'firewall':
```

Comment:
I wonder if this should be made configurable. Many users may want to purge unknown rules. Also I don't think it defaults to purging so maybe leave it out altogether?